src/Security/Authentication/AuthenticationFailureHandler.php

```diff
@@ -91,7 +91,7 @@ public function onAuthenticationFailure(Request $request, AuthenticationExceptio
     private function determineTargetUrl(Request $request): string
     {
         if ($this->scopeMatcher->isBackendRequest($request)) {
-            return $request->getRequestUri();
+            return 'contao_backend_login';
```

Comment:
This looks wrong to me. The method should return a path, shouldn't it?

Reply:
HttpUtils will use the UrlGenerator if it's a route name.